app/src/main/res/values/dimens.xml

```diff
@@ -9,6 +9,7 @@
     <dimen name="media_card_text_padding_vertical">10dp</dimen>
 
     <dimen name="navigation_bar_height">65dp</dimen>
+    <dimen name="navigation_rail_width">80dp</dimen>
```

Comment:
Why do we define sizes for these anyways

Reply:
Because we're not using a Scaffold :kappa:

Reply:
Scaffold doesn't constrain the size either, size is set by the component

Reply:
How would I add Navigation Bar padding to the other components?

Reply:
Idk, stop using a box

Reply:
o7